infra/modules/providers/azure/app-service/README.md

```diff
@@ -0,0 +1,34 @@
+## Azure App Service
+
+Build, deploy, and scale enterprise-grade web, mobile, and API apps running on any platform. Meet rigorous performance, scalability, security and compliance requirements while using a fully managed platform to perform infrastructure maintenance.
+
+More information for Azure App Services can be found [here](https://azure.microsoft.com/en-us/services/app-service/)
+
+Cobalt gives ability to specify following settings for App Service based on the requirements:
+- name : The name of the App Service.
+- resource_group_name : The Name of the Resource Group where the App Service exists.
+- location : The Azure location where the App Service exists.
+- app_service_plan_id : The ID of the App Service Plan within which the App Service exists.
+- tags : A mapping of tags to assign to the resource.
+
+Please click the [link](https://www.terraform.io/docs/providers/azurerm/d/app_service.html) to get additional details on settings in Terraform for Azure App Service.
+
+## Usage
+
+```
+variable "name" {
+  default = "prod"
+}
+
+variable "location" {
+  default = "eastus"
+}
+
+resource "azurerm_app_service" "appsvc" {
```

Comment:
as mentioned in previous comments, the documented usage should be aligned with how we're using these modules in our own templates. That being said, we should reference our own modules as terraform modules.

Reply:
The module is being defined here. And the module usage is defined below.

Reply:
the goal of these modules is to avoid developers from needing to define resource like this `resource "azurerm_app_service" "appsvc"`

Reply:
But the actual usage is after the definition. A couple more lines down, the example usage specifies how it has to be used.

Reply:
my comment was not addressed. The documentation seems a bit confusing as this section represents how a developer would use the `app_service` module. Can you please remove the module definition from the readme and reference the reader to the location of the definition.

Reply:
Here's an usage documentation example for how the load balancer module is defined https://github.com/Azure/terraform-azurerm-loadbalancer